lerna.json

```
@@ -0,0 +1,5 @@
+{
+    "$schema": "node_modules/lerna/schemas/lerna-schema.json",
+    "useWorkspaces": true,
```

```suggestion
    "npmClient": "yarn",
    "useWorkspaces": true,
```

This [seems to](https://www.honeybadger.io/blog/monorepo-yarn-workspace-lerna/) make lerna use yarn, have you tried it?